src/Security/Voter/HandleCommandVoter.php

```diff
@@ -64,13 +64,15 @@ protected function supports($attribute, $subject)
     protected function voteOnAttribute($attribute, $subject, TokenInterface $token)
     {
         $allowedRoles = $this->getAllowedRoles(get_class($subject));
+        $actualRoles = $this->roleHierarchy->getReachableRoles($token->getRoles());
 
-        if (count($allowedRoles) > 0) {
-            return $this->decisionManager->decide($token, $allowedRoles);
-        } else {
-            // default conclusion is access denied
-            return false;
+        foreach ($actualRoles as $role) {
+            if (in_array($role->getRole(), $allowedRoles)) {
+                return true;
+            }
         }
```

Comment:
This will change the behavior if the access decision strategy is set to "unanimous": https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php#L150